File: Programación y Algoritmos/Actividad 18/areayperimetro.py
import math


def CalcularPerimetro(radio: float) -> float:
    perimetro = (radio*2) * math.pi


    return perimetro


def CalcularArea(radio: float ) -> float:
    area = (radio*radio)*math.pi

    return area


while True: 
    try:
        radio = float(input("Introduzca el radio del círculo del cuál quiera realizar los cálculos:  "))
        break
    except ValueError: 
        print("Ese número no es válido, introduzca uno nuevo ")
print("El área del círculo de radio " + str(radio) + " es de: " + str(CalcularArea(radio)))
print("El perímetro del círculo de radio " + str(radio) + "es de: " + str(CalcularPerimetro(radio)))

File: Programación y Algoritmos/Actividad 18/areayperimetro.cpp
#define _USE_MATH_DEFINES
#include <iostream>
#include <cmath>

using namespace std;

float CalcularArea(float radio)
{
    return (pow(radio,2)) * M_PI;
}

float CalcularPerimetro(float radio)
{
    return(radio*2*M_PI);
}

int main()
{
    int radio;

    cout << "Introduzca el radio del circulo del cual quiera obtener area y perimetro" << endl; 
    cin >> radio;
    cout << endl; 

    cout << "El area del circulo de radio " << radio << " es de " << CalcularArea(radio) << endl;
    cout << "El perimetro del circulo de radio " << radio << " es de " << CalcularPerimetro(radio) << endl;


    
}

